web/components/admin/Toolbar/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Select } from '../../input/select';
+
+class Toolbar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.collectionSelectLabels = {
+      hacker: { value: 'hacker', label: 'Hacker' },
+      mentor: { value: 'mentor', label: 'Mentor' },
+      volunteer: { value: 'volunteer', label: 'Volunteer' },
+    };
+  }
+
+  render() {
+    const { applicantType, switchApplicantType, className } = this.props;
+    return (
+      <div className={`toolbar fill-width ${className}`}>
+        <Select
+          onChange={option => switchApplicantType(option.value)}
+          label="Form:"
+          name="collection-input"
+          value={this.collectionSelectLabels[applicantType].label}
```

Comment:
I realized this functionality isn't working properly. If you select an option other than hacker, the dropdown doesn't highlight the current option.

This is one fix:
![image](https://user-images.githubusercontent.com/21287547/49139044-a4033780-f2a5-11e8-918a-e5413761ef6b.png)

I'm ok if you address this in another pr

Reply:
What browser are you using? It's working on my end in Chrome and Firefox, value is passed as a string. 

Or are you suggesting to do the object check just in case an object is passed into value?

Reply:
i'm using chrome. btw it still works, just that when "volunteer" is selected, when you open the dropdown, "hacker" is highlighted. 

I'm suggesting we pass in an object as the `value` in `Select`, and modify the `Select` component to allow objects to be passed in to `value`

Reply:
Ohhh sorry I thought you were talking about the placeholder text, I'll fix this